### ISSUE TITLE
Poll : Multiple vote isn't possible

### ISSUE THREAD

**Issue by rich20** [Kunena/Kunena-Forum#5089]
#### Steps to reproduce the issue

Setting:
Single Vote Only = No
The rest is default setting

#### Expected result

User can vote again after 15 min

#### Actual result

Only a single vote is possible

#### System information (as much as possible)

Joomla version: 3.6.4 and 3.7.0-dev
Kunena version: 5.0.4.1 and 5.0.5-dev
Php version: 7.0.10
Database version:

**Comment by xillibit** [comment]
At the begining the poll feature hasn't been designed to support multiple votes

**Comment by rich20** [comment]
Yes, but now we have this option in the configuration. Therefore it should works.

**Comment by xillibit** [comment]
I will look to make working the multiple votes

**Comment by 810** [comment]
@xillibit could you check this one
